searchcore/src/tests/proton/matching/query_test.cpp

```diff
@@ -164,6 +164,17 @@ fef_test::IndexEnvironment plain_index_env;
 fef_test::IndexEnvironment resolved_index_env;
 fef_test::IndexEnvironment attribute_index_env;
 
+vespalib::string
+termAsString(const search::query::Range &term) {
```

Comment:
Why are these here? (to avoid updating the test to use the new API?)

Reply:
Only used in test.